lib/algo/DMD.cpp

```diff
@@ -56,12 +58,14 @@ DMD::DMD(int dim)
     MPI_Comm_rank(MPI_COMM_WORLD, &d_rank);
     MPI_Comm_size(MPI_COMM_WORLD, &d_num_procs);
     d_dim = dim;
+    d_state_offset = state_offset;
     d_trained = false;
     d_init_projected = false;
 }
 
-DMD::DMD(int dim, double dt)
+DMD::DMD(int dim, double dt, Vector* state_offset)
 {
+    // Vanilla DMD
```

Comment:
Although we know what "vanilla" means here, users may not. Can you make a more descriptive comment that this is the basic DMD with uniform steps?

Reply:
The comments are modified and now put in the header file.